app/views/report_templates/generate.html.erb

```diff
@@ -31,3 +31,21 @@
 
   <%= submit_or_cancel f, false, { data: { disable_with: false } } %>
 <% end %>
+
+<script type="text/javascript">
```

Comment:
why is this part of the view vs under webpack?

Reply:
I also wonder if we could move it out :-)

Reply:
I've contained it in the webpack file initialize and just calling its function.